src/main/java/service/CalculateFruit.java

```diff
@@ -0,0 +1,10 @@
+package service;
+
+import java.util.List;
+import java.util.Map;
+import model.FruitOperation;
+import model.FruitType;
+
+public interface CalculateFruit {
+    Map<FruitType, Integer> calculateFruit(List<FruitOperation> listFruitOperation);
```

Comment:
better to use directly fruit String representation rather enum
```suggestion
    Map<String, Integer> calculateFruits(List<FruitTransaction> fruits);
```